packages/eslint-plugin/docs/rules/no-implicit-any-catch.md

```diff
@@ -6,6 +6,12 @@ By default, TypeScript will type a catch clause variable as `any`, so explicitly
 
 The `noImplicitAny` flag in TypeScript does not cover this for backwards compatibility reasons.
 
+## Warning
+
+With TypeScript 4.4 starts defaulting to the `unknown` Type in Catch Variables
```

Comment:
It does not **default** to it - I think you misunderstood the doc.
It's a new option that is behind a flag.